### PR TITLE
feat: add trivy misconfig, secret, license dtos

### PR DESCRIPTION
**Pull request by hutomosaleh**
This PR adds the DTOs needed for [misconfigurations](https://trivy.dev/v0.57/docs/scanner/misconfiguration/), [licenses](https://trivy.dev/v0.57/docs/scanner/license/), and [secrets](https://trivy.dev/v0.57/docs/scanner/secret/) scans. It is needed so that the pod manager can send a complete container scan from Trivy.

### CODE REVIEW

File: model/src/main/kotlin/de/fraunhofer/iem/spha/model/adapter/trivy/TrivyDto.kt
Comment (janniclas) [inline model/src/main/kotlin/de/fraunhofer/iem/spha/model/adapter/trivy/TrivyDto.kt:32]: could you please quickly check whether we use default values in our other data classes? If I remember correctly, we opted to not use default values at all and leave it to the library's users to decide how to do the initialization, but I'm not 100% from the top of my head :)

**Review by janniclas — APPROVED** [review]
The PR looks good to me except the question regarding the default values. If we use default values in other data models you can directly merge this PR, if not please just remove the initialization with `=listOf()` and merge afterwards

File: model/src/main/kotlin/de/fraunhofer/iem/spha/model/adapter/trivy/TrivyDto.kt
Comment (hutomosaleh) [reply model/src/main/kotlin/de/fraunhofer/iem/spha/model/adapter/trivy/TrivyDto.kt:32]: Defaults values are used in the tlc's [project dto](https://github.com/fraunhofer-iem/spha/blob/main/model/src/main/kotlin/de/fraunhofer/iem/spha/model/adapter/tlc/ProjectDto.kt). The other dtos does not have default values. If we're going to make all dtos not have default values, I think its best to make another PR, since tests needs to be changed too.

Comment (janniclas) [reply model/src/main/kotlin/de/fraunhofer/iem/spha/model/adapter/trivy/TrivyDto.kt:32]: agreed. I think it would be good if we also change the tlc's dtos. So let's merge this and do a dedicated PR  as you suggested

**Review by janniclas — APPROVED** [review]

**Pull request merged**
